### PR TITLE
Release 2.9.2 notes and version bump

### PR DESCRIPTION
**Pull request by peterbarker**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,31 @@
...
+* add a still-waiting callback for connect() to provide debug on where the connection is up to
+* several new tests added (including, play_tune, reboot and set_attitude_target)
+
+### Cleanup
+* flake8 compliance improvements
+* test includes pruned
+* examples cleaned up
+
+### Bug Fixes
+* ignore GCS heartbeats for heartbeats
```
Comment (hamishwillee) [inline CHANGELOG.md]: Don't know what this means.

**Comment by hamishwillee** [comment]
Awesome. Also going on PyPi? I'm happy with this! Do you need anything from me?

File: CHANGELOG.md (hunk 1 above)
Comment (peterbarker) [reply CHANGELOG.md]: It means Peter was tired.  Fixed.

**Comment by peterbarker** [comment]
@hamishwillee Yes, that's what "release" really means :-)

At the point I plan on doing the release on Monday.  Hopefully we get more "yays" before then.  And maybe some offers of help-with-support :-)

**Comment by hamishwillee** [comment]
Yay!!!!!
I've posted on gitter to let more people know about this!

**Comment by amilcarlucas** [comment]
To have full python3 support this probably requires an updated pymavlink?

**Comment by peterbarker** [comment]
On Fri, 15 Mar 2019, Amilcar Lucas wrote:

> To have full python3 support this probably requires an updated pymavlink?

You're suggesting we bump the pymavlink version required?

Perhaps we use the same one that MAVProxy currently uses?

**Comment by amilcarlucas** [comment]
yes, that is what I meant.
And why not the latest stable pymavlink version?

**Comment by peterbarker** [comment]
@amilcarlucas I don't see anything in the way of Python3 stuff between 2.2.0 and 2.3.4

I *do* see the new multi-vehicle stuff.  I'd rather not introduce that stuff into the mix at this point :-)

IOW - I can't see any *requirement* to have a more recent pymavlink.

**Comment by amilcarlucas** [comment]
Yes, you are correct. I was thinking about https://github.com/ArduPilot/pymavlink/commit/c16da2f26b2d0f19d692e870bbe589e7a39bb6b1 but that is already included in 2.2.0

**Comment by peterbarker** [comment]
I've merged all of this in.  We have a 2.9.2 release.

**Pull request closed without merging**